Extension/src/LanguageServer/extension.ts

```diff
@@ -527,6 +528,10 @@ function onResetDatabase(): void {
     clients.ActiveClient.resetDatabase();
 }
 
+function selectDefaultCompiler(): void {
+    clients.ActiveClient.promptSelectCompiler(true);
+}
+
 function onSelectConfiguration(): void {
     if (!isFolderOpen()) {
         vscode.window.showInformationMessage(localize("configuration.select.first", 'Open a folder first to select a configuration'));
```

Comment:
Can you add a period to this message too, even though it's a pre-existing issue?